### PR TITLE
Deprecate the Proofview.V82 module

### PR DESCRIPTION
**Pull request by ppedrot**
The end of the tunnel is near.

### CODE REVIEW

**Comment by ppedrot** [comment]
Oh, this stupid deprecated module warning issue again. What's the correct magic incantation this time? I am not opening it anywhere.

File: proofs/tacmach.mli [hunk 1]
```diff
@@ -21,7 +21,7 @@ open Evd
 module Old :
 sig
 
-type tactic = Proofview.V82.tac
+type tactic = Proofview.V82.tac [@@ocaml.warning "-3"]
```
Comment (SkySkimmer) [inline proofs/tacmach.mli:24]:
```suggestion
[@@@ocaml.warning "-3"] 
type tactic = Proofview.V82.tac
[@@@ocaml.warning "+3"]
```
same in the .ml

Comment (SkySkimmer) [reply proofs/tacmach.mli:24]: Or just inline the definition

**Review by Alizter — APPROVED** [review]

**Comment by Alizter** [comment]
I can merge tomorrow

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**